Essentials/powershell.yml

```diff
@@ -0,0 +1,49 @@
+Name: powershell
+Description: PowerShell Wrapper For Wine
+Provider: ProjectSynchro
+License: GPLv2
+License_url: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+Dependencies:
+- powershell_core
+Steps:
+- action: copy_file
+  file_name: 'null'
```

Comment:
is this necessary?

Reply:
I think so... yes, because that way can save both versions(32 and 64, both need the same name) of the executable... and by version.

Reply:
The `copy_file` action with `null` is only to create a directory structure.
```
pswrapper-*version*/
pswrapper-*version*/32/
pswrapper-*version*/64/
```

Reply:
oh ok fine